internal/entity/metrics/counter.go

```diff
@@ -0,0 +1,30 @@
+package metrics
+
+import (
+	"errors"
+	"strconv"
+)
+
+type Counter struct {
+	list map[string]float64
```

Comment:
It would be better if you rename this name.

Reply:
++